content/blog/2018-12-28-subquery.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Subquery
```

Comment:
Just "Subquery" as single singular noun is an odd title. How about "Subquery Support".